web/src/connectors/documents-page-control-connector/use-notify-processing.ts

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from 'react';
+import { NotificationTypes, showNotification } from 'shared/components/notifications';
+
+const notifyProcessing = () => {
+    showNotification({
+        type: NotificationTypes.hint,
+        text: 'Your file is still processing. This may take a while'
+    });
+};
+
+export const useNotifyProcessing = (isUploaded: boolean) => {
```

Comment:
In case of failed request - will this still work?

Reply:
The notification of request error is done separately from this hook, please check new changes